app/services/eyeshade/referral_totals.rb

```diff
@@ -0,0 +1,27 @@
+class Eyeshade::ReferralTotals < Eyeshade::BaseApiClient
+  def perform
+    if Rails.application.secrets[:api_eyeshade_offline]
+      result = perform_offline
+    else
+      begin
```

Comment:
what is your reasoning for the `begin` block?

Reply:
Probably just copy pasted. I'll add a catch block